### ISSUE TITLE
Conversion J2N des liens

### ISSUE THREAD

**Issue by DuzzAtNemo** [Julien-Arlandis/phpNemoServer#5]
- La balise [­a]... [/a] est convertie en URL **sans** crochets **<...>**.
- Le lien vers une image téléchargée sur le serveur, est converti en URL **avec** crochets.
- Le lien vers une image insérée depuis une URL, est converti en URL **sans** crochets.

A titre d'illustration, voir l'article **`<news:nHNAAXZm0LsxN0_0MdkOH2g2XVI@jntp>`** avec un client NNTP.

Il est souhaitable d'uniformiser ces conversions, de préférence en URL **avec** crochets.

**Comment by yamo-nntp** [comment]
Oui bonne idée. Mais c'était le cas sur news? 
http://al.howardknight.net/msgid.cgi?STYPE=msgid&A=0&MSGI=%3CnHNAAXZm0LsxN0_0MdkOH2g2XVI%40jntp%3E

**Comment by DuzzAtNemo** [comment]
Non, les conversions J2N sont actuellement les mêmes sur 'news'.
